paymentsheet-example/src/androidTest/java/com/stripe/android/screenshot/TestPaymentSheetScreenshots.kt

```diff
@@ -236,6 +240,25 @@ internal class TestPaymentSheetScreenshots : BasePlaygroundTest(disableAnimation
         )
     }
 
+    @Test
+    fun testPaymentSheetMandate() {
```

Comment:
I'm fine with adding this, but my hope is that it won't be needed very soon. I plan to test form building much more extensively using paparazzi (for all payment methods), and using multiple variants, ie, not showing the mandate when it shouldn't be shown for payment methods that only require the mandate during setup.

Reply:
Would the Paparazzi tests be able to test the whole the `SelectedSavedPaymentMethods` flow with the LPMs following your changes? From my understanding, #ir-universe-locus occurred as a result of an issue with fragmented LPM logic within `PaymentSheet`.